src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Et3ResponseHelper.java

```diff
@@ -17,9 +22,14 @@
     "PMD.ExcessiveMethodLength", "PMD.ClassNamingConventions", "PMD.PrematureDeclaration"})
 public class Et3ResponseHelper {
 
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+    private static final String TEMPLATE_NAME = "ET3_0922 (1).docx";
+    private static final String OUTPUT_NAME = "ET3 Response.pdf";
     private static final String CLAIMANT_NAME_TABLE = "<pre> ET1 claimant name&#09&#09&#09&#09 %s</pre><hr>";
     public static final String START_DATE_MUST_BE_IN_THE_PAST = "Start date must be in the past";
     public static final String END_DATE_MUST_BE_AFTER_THE_START_DATE = "End date must be after the start date";
+    public static final String CHECKED = "■";
+    public static final String UNCHECKED = "□";
```

Comment:
Unusual characters here that are worth a comment on the right with the unicode character code for developer clarity